includes/fbs.availability.inc

```diff
@@ -80,6 +80,7 @@ function fbs_availability_holdings($provider_ids) {
       // And FBS can't tell us this.
       'reserved_count' => 0,
       'is_periodical' => FALSE,
+      'reservation_count' => $item->reservations
```

Comment:
Well then I think you should remove reserved_count - makes no sense to keep both variables.
```
$ grep -r 'reserved_count' *
ding2/includes/fbs.availability.inc:      'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:   *   reserved_count: # int
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
ding2/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/includes/fbs.availability.inc:      'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:   *   reserved_count: # int
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
fbs/tests/AvailibilityProviderTest.php:        'reserved_count' => 0,
openruth/openruth.availability.inc:        $h['reserved_count'] += $itemHolding->ordersCount;
openruth/openruth.availability.inc:    'reserved_count' => isset($holding->ordersCount) ? (int) $holding->ordersCount : 0,
openruth/openruth.availability.inc:    $row['data']['Copies'] = array('data' => $h['reserved_count'], 'colspan' => 3);

$ grep -r 'reservation_count' *
ding_user_frontend/ding_user_frontend.module:      'variables' => array('loan_count' => NULL, 'reservation_count' => NULL, 'debt_count' => NULL),
ding_user_frontend/templates/ding_user_frontend_user_status.tpl.php:  <span class="amount">(<?php print $reservation_count; ?>)</span>
fbs/includes/fbs.availability.inc:  $reservation_count = (int) $res['reservation_count'];
fbs/includes/fbs.availability.inc:  $result = _fbs_set_table_html($res['holdings'], $reservation_count);
fbs/includes/fbs.availability.inc:  $reservation_count = 0;
fbs/includes/fbs.availability.inc:    $reservation_count += (int) $records['reservation_count'];
fbs/includes/fbs.availability.inc:  $result = _fbs_set_table_html($holdings, $reservation_count);
```